mysql-test/include/mtr_check.sql

```diff
@@ -206,21 +206,28 @@ BEGIN
   -- not give consistent result either.
   --
   SELECT /*+SET_VAR(use_secondary_engine=OFF)*/ USER, HOST, DB, COMMAND, INFO FROM INFORMATION_SCHEMA.PROCESSLIST
-    WHERE COMMAND NOT IN ('Sleep', 'Daemon', 'Killed')
+    WHERE COMMAND NOT IN ('Sleep', 'Daemon')
```

Comment:
It seems that "Killed" state was added to this condition by Upstream's fix. Should we really revert this?

Reply:
Good catch. I have changed it by mistake. Fixed.